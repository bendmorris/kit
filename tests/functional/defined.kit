function main() {
    var b = defined stdout;
    puts(if b then "true" else "false");

    static if defined stdout {
        puts("stdout exists");
    } else {
        puts("stdout doesn't exist");
    }
    static if defined stdout123987 {
        puts("stdout123987 exists");
    } else {
        puts("stdout123987 doesn't exist");
    }
}
