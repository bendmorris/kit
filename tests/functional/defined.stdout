true
stdout exists
stdout123987 doesn't exist
